tests/queries/0_stateless/01129_dict_get_join_lose_constness.sql

```diff
@@ -1,8 +1,6 @@
--- Tags: no-parallel
+DROP DICTIONARY IF EXISTS system.dict_01129;
 
-DROP DICTIONARY IF EXISTS system.dict1;
-
-CREATE DICTIONARY IF NOT EXISTS system.dict1
```

Comment:
We probably should not be creating thrash under `system` to begin with (unless we are testing creating trash under system explicitly).